app/templates/secrets.hbs

```diff
@@ -9,7 +9,7 @@
         <span class="icon icon-menu">menu</span>
       {{/link-to}}
       <form action="#">
-        {{input placeholder="Search..." value=query}}
+        {{input id="search" placeholder="Search..." value=query}}
```

Comment:
That id is dynamic. I suggest you to take a look at how I do it at the 'secret-revealer' view. Hint: use a different selector, not the id.